From b0ea9eff08065611915b1c14a044281309cb19c3 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Fri, 5 Mar 2021 16:55:18 +0800
Subject: [PATCH] Follow upstream instructions to include system pybind11
 (#566)

The previous way doesn't work with Arch Linux's pybind11, but the official recommended way does.

Ref: https://pybind11.readthedocs.io/en/latest/cmake/index.html
---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d4b6530..cfc02c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -219,9 +219,7 @@ endif()
 
 if (BUILD_PYTHON)
   if(USE_SYSTEM_PYBIND11)
-    include(pybind11Config)
-    include(pybind11Common)
-    include(pybind11Tools)
+    find_package(pybind11 CONFIG)
   else()
     add_subdirectory(deps/pybind11-2.5.0)
   endif()
-- 
2.30.2

